cpp/lib/src/ZoteroHarvesterConversion.cc

```diff
@@ -391,6 +391,12 @@ void StripBlacklistedTokensFromAuthorName(std::string * const first_name, std::s
 }
 
 
+bool Is655Keyword(const std::string &keyword) {
+    static const auto keyword_list(FileUtil::ReadLines::ReadOrDie(UBTools::GetTuelibPath() + "zotero-enhancement-maps/keywords_as_655.txt"));
```

Comment:
That seems like a strange name "keywords_as_655.txt".

Reply:
What would be a better name?

Reply:
marc655_keywords.txt?